src/sdk-reference/cpp/1/websocket/close/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: sdk.html.hbs
+title: close
+description: Disconnect the SDK
+---
+
+# close
+
+Closes the current connection to Kuzzle.  
+
+## Signature
+
+```cpp
+std::string close();
```

Comment:
Why does this method return a string? And if that's not an oversight,  then the `## Return` chapter is missing.